src/styles/_catalog.scss

```diff
@@ -14,11 +14,11 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-    transition: 300ms;
+    transition: all 300ms;
```

Comment:
never add transition for `all` properties. You need add transition just for color and transform